plugins/janus_videoroom.c

```diff
@@ -2321,6 +2330,43 @@ void janus_videoroom_hangup_media(janus_plugin_session *handle) {
 	}
 }
 
+static void janus_videoroom_sdp_v_format_vp8(char *mline, int mline_size, int pt, int b, const char *video_mode) {
```

Comment:
I see you're right, it does warn for mixed, I haven't used it in a while (though it does work ;)

```
$ gcc -Wall -o test test.c
test.c:4:35: warning: cannot mix positional and non-positional arguments in format string [-Wformat]
        printf("count=%d cmd=%s again=%1$d reprise=%1$d\n", argc, argv[0]);
                                      ~~~^
1 warning generated.
$ ./test two three four
count=4 cmd=./test again=4 reprise=4
```

Anyway I think all positional in these format strings isn't too ugly.